test/functional/fixtures/api/es-next/right-click/test.js

```diff
@@ -0,0 +1,61 @@
+var expect = require('chai').expect;
+
+// NOTE: we run tests in chrome only, because we mainly test server API functionality.
+// Actions functionality is tested in lower-level raw API.
+describe('[API] t.rightClick()', function () {
+    it('Should make right click on a button [ONLY:chrome]', function () {
+        return runTests('./testcafe-fixtures/right-click-test.js', 'Right click button', { shouldFail: true })
+            .catch(function (err) {
+                expect(err).to.contains('Right click on the button');
+                expect(err).to.contains(
+                    ' 10 |test(\'Incorrect action option\', async t => {' +
+                    ' 11 |    await t.rightClick(\'#button\', { offsetX: -3 });' +
+                    ' 12 |});' +
+                    ' 13 |' +
+                    ' 14 |test(\'Right click button\', async t => {' +
+                    ' > 15 |    await t.rightClick(\'#button\');' +
```

Comment:
You can reduce callsite checks just to the exact string. Callsite are already heavily tested, so we need just to check that we get the correct one, so it could be just:
```js
expect(err).to.contains(' > 15 |    await t.rightClick(\'#button\');');
```